docs(QRCode): Revamp Overview (#3664)
telerik/blazor-docs: components/barcodes/qrcode/overview.md

## QRCode Parameters

The Blazor Barcode provides various parameters that allow you to configure the component. Also see the [Telerik QRCode API reference](slug:telerik.blazor.components.telerikqrcode).

@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)

| Parameter | Type | Description |
| --- | --- | --- |
| `Background` | `string` | The background color of the QRCode. Accepts a valid CSS color string, including HEX and RGB. |
| `Class` | `string` | The CSS class that will be rendered on the main wrapping element of the QRCode component. |
| `Color` | `string` | Defines the color of the QRCode elements. The default color is black. |
| `Height` | `string` | Sets the height of the QRCode. If `Height` is set and the `Size` property is not set, the same value as `Height` should be set to `Width`. |
| `Padding` | `double` | Defines the minimum distance in pixels that should be left between the border and the QR modules. |
| `Encoding` | `QRCodeEncoding` `enum` | The encoding mode used to encode the value. |
| `ErrorCorrection` | `QRCodeErrorCorrectionLevel` `enum` | The error correction level used to encode the value. |

> | `RenderAs` | `RenderingMode` enum <br /> `Svg` | Defines the preferred rendering mode - svg/canvas. |

Reviewer: ```suggestion
| `RenderAs` | `RenderingMode` enum <br /> `Svg` | Defines the preferred rendering mode&mdash;SVG or canvas. |
```

> | `Size` | `string` | Specifies the size (`Width` and `Height`) of a QR code in pixels (i.e. "200px") as the QRCode is a square. You can read more details for the dimension properties in the [Dimensions article](slug:common-features/dimensions). Setting both `Size` and `Width` or `Height` will throw an error. Setting different values to `Width` and `Height` will also cause an issue. To set an optimal `Size`, consider the expected scanning distance and data capacity. |

Reviewer: ```suggestion
| `Size` | `string` | Specifies the size (`Width` and `Height`) of a QR code in pixels (i.e. "200px") as the QRCode is a square. You can read more about the dimension properties in the [Dimensions article](slug:common-features/dimensions). Setting both `Size` and `Width` or `Height` will throw an error. Setting different values to `Width` and `Height` will also cause an issue. To set an optimal `Size`, consider the expected scanning distance and data capacity. |
```

| `Value` | `string` | Defines the initial value of the QRCode. |
| `Width` | `string` | Sets the width of the QRCode. If `Height` is set and the `Size` property is not set, the same value as `Width` should be set to `Height`. |

### QRCodeOverlay parameters

The nested `QRCodeOverlay` tag exposes parameters that allow you to choose from predefined layouts or insert a custom image in the Blazor QRCode:

| Parameter | Type | Description |
| ----------- | ----------- | ----------- |

> | `Type` | `QRCodeOverlayType` `enum` | The available options are `Image` and `Swiss`. When set to image, you have to specify the URL source of the image. |

Author: ```suggestion
| `Type` | `QRCodeOverlayType` `enum` | The available options are `Image` and `Swiss`. When set to `Image`, you must set the `ImageUrl` parameter too. |
```

> | `Type` | `QRCodeOverlayType` `enum` | Available options are `Image` and `Swiss`. When set to image, you have to specify the url source of the image. If set to swiss, a [Swiss QR Code](https://blog.xsuite.com/en/swiss-qr-code#:~:text=This%20QR%20code%20is%20called,(e.g.%20in%20PDF%20format).) is created. |

Reviewer: ```suggestion
| `Type` | `QRCodeOverlayType` `enum` | The available options are `Image` and `Swiss`. When set to image, you have to specify the URL source of the image. |
```
